addressbook/interface/exchangeAutoCompleteResult/mivExchangeAutoCompleteResult.js

```diff
@@ -18,21 +18,19 @@
  *
  * ***** BEGIN LICENSE BLOCK *****/
 
-var Cc = Components.classes;
 var Ci = Components.interfaces;
 var Cu = Components.utils;
-var Cr = Components.results;
-var components = Components;
 
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 
 Cu.import("resource:///modules/mailServices.js");
 
 function mivExchangeAutoCompleteResult() {
-
-    //dump("mivExchangeAutoCompleteResult init\n");
-    this._cards = new Array();
+    // _cards contain all Cards in value and are fetchable by localid as key
+    this._cards = new Object();
+    // _idcards keeps localids sorted by arrive
```

Comment:
Typo or spelling issue with `arrive`.

Reply:
Indeed, that's a typo, I wanted to say "by arrival", I'm sending a patch to fix it.

```suggestion
    // _idcards keeps localids sorted by order of arrival
```